### PR TITLE
Bump version to 1.7.8.3

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           |  1.7.8.x
| Description?      | Bump version to 1.7.8.3
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A
| Possible impacts? | N/A

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26630)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
You forget to change: AppKernel::RELEASE_VERSION

File: app/AppKernel.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 class AppKernel extends Kernel
 {
-    const VERSION = '1.7.8.1';
+    const VERSION = '1.7.8.2';
```
Comment (jolelievre) [inline app/AppKernel.php]: Missing update on `RELEASE_VERSION`

**Comment by Progi1984** [comment]
@PierreRambaud @jolelievre @matks Fixed & Rebased

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
It should be 1.7.8.2 :thinking:

**Comment by Progi1984** [comment]
@PierreRambaud No, the 1782 in on `1.7.8.x-build` by @matks. The next version in `1.7.8.x` will be 1783.

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @Progi1984!

**Pull request merged**